classes/GroupReduction.php

```diff
@@ -285,4 +285,12 @@ public static function deleteCategory($id_category)
 
         return true;
     }
+
+    /**
+     * Reset static cache (mainly for test environment)
+     */
+    public static function resetStaticCache()
+    {
+        GroupReduction::$reduction_cache = [];
```

Comment:
```suggestion
        static::$reduction_cache = [];
```

Reply:
Could you also add this `GroupReduction::resetStaticCache()` in `CommonFeatureContext::clearCache`

Reply:
already done